packages/e2e-tests/jest.config.js

```diff
@@ -12,6 +12,6 @@ module.exports = {
 	testPathIgnorePatterns: [
 		'/node_modules/',
 		'/wordpress/',
-		'e2e-tests/specs/performance.test.js',
+		'e2e-tests/specs/performance/',
```

Comment:
You probably can move:
https://github.com/WordPress/gutenberg/blob/7e9df78eaa77c94ce5228695445868ce8f2757fd/packages/e2e-tests/specs/.gitignore to the `performance` folder.